.github/workflows/test_proto.yml

```diff
@@ -0,0 +1,28 @@
+# docs: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CT ProtoBuf
+
+on:
+  push:
+    branches: ['master', 'main']
```

Comment:
There is no `main` branch here.

Reply:
yep. in case we ever go with the name change of the default branch from `master` to `main`, we are prepared.